src/main/java/no/sikt/sws/WorkspaceIndexHandler.java

```diff
@@ -30,11 +30,10 @@ protected IndexResponse processInput(Void input, RequestInfo request, Context co
         var workspace = RequestUtil.getWorkspace(request);
         var index = request.getPathParameter(RESOURCE_IDENTIFIER);
 
-        var restClientOpenSearch = new OpenSearchClient();
```

Comment:
You replaced this with
`public OpenSearchClient openSearchClient `

but how do you initialise it?